server/cli/launch.py

```diff
@@ -354,7 +352,7 @@ def launch(
     # > cellxgene launch --dataroot <url>
 
     if dump_default_config:
-        print(default_config)
+        print(DEFAULT_CONFIG.default_config)
```

Comment:
cellxgene launch --dump-default-config

should print out yaml, so that users can then modify.  
DEFAULT_CONFIG.default_config is a python dictionary.

Maybe we should also have a test for this?